oml_metafeature_map.json

```diff
@@ -0,0 +1,45 @@
+{
+    "EquivalentNumberOfAtts": ["EquivalentNumberOfFeatures", 1],
+    "MeanCardinalityOfNominalAttributes": ["MeanCardinalityOfNominalFeatures", 1],
```

Comment:
Make dict into dict of dicts following a pattern like this:
"EquivalentNumberOfAtts": {
        "oml_name": "EquivalentNumberOfFeatures",
        "scale_factor": 1,
        // "margin_of_error": "default"
        // {
        //     "function_name": "percent_error",
        //     "tolerance": 0.05
        // }
    },